config.go

```diff
@@ -42,6 +42,11 @@ type ConfigSection interface {
 	ValidateAndDefault() error
 }
 
+type KeyValuePair struct {
+	Key   string      `bson:"key" json:"key" yaml:"key"`
+	Value interface{} `bson:"value" json:"value" yaml:"value"`
+}
```

Comment:
Does it make sense to move this to the util package? 

Also I think it'd be handy/sensible if the convert functions were methods on the 

`type KeyValuePairSlice []KeyValuePair`

Reply:
I can't move it to the util package since it would create an import cycle (evergreen package tests import testutil which imports util). I moved it to a separate file though